CVLByExample/ReadOnlyReentrancy/certora/conf/BuiltinROReentrancyVulnerableBankOther.conf

```diff
@@ -0,0 +1,9 @@
+{
```

Comment:
What does "other" mean in the name. Can you give a more explanatory name?

Reply:
yeah I have an idea